verify.js

```diff
@@ -116,6 +129,7 @@ const parseConfig = (config) => {
   const workingDir = config.working_directory
   const contractsBuildDir = config.contracts_build_directory
   const optimizerSettings = config.compilers.solc.settings.optimizer
+  const verifyPreamble = config.verify.preamble || "";
```

Comment:
Will this throw if config.verify is undefined?

Reply:
If it's undefined the or section (`||`) will be triggered setting it to an empty string by default. But since we actually check for the existence of `verifyPreamble` beforehand, it's technically okay to not set the default. Just makes typing checking a bit easier because it will always be a string.

Reply:
Good catch. Fixed in 14be3c.